Respect the autotools docdir switch. Avoid installing a copy of the license.

--- doc/Makefile.am
+++ doc/Makefile.am
@@ -1,10 +1,9 @@
 man3dir  = $(mandir)/man3
-docdir   = $(datadir)/doc/$(PACKAGE)-$(VERSION)
+docdir   = @docdir@
 docfiles = \
 	$(top_srcdir)/README \
 	$(top_srcdir)/AUTHORS \
 	$(top_srcdir)/NEWS \
-	$(top_srcdir)/COPYING \
 	$(top_srcdir)/ChangeLog
 
 targets = doxygen.stamp
